calm/dsl/cli/utils.py

```diff
@@ -31,6 +32,22 @@ def highlight_text(text, **kwargs):
     return click.style("{}".format(text), fg="blue", bold=False, **kwargs)
 
 
+def read_file(file_path):
```

Comment:
These methods are already defined in calm/dsl/builtins/models/utils.py.

Use _from calm.dsl.builtins import read_file_ to use these methods